week-1-ts-ecosystem/w01e04-light-me-up/package.json

```diff
@@ -1,5 +1,13 @@
 {
   "name": "w01e04-light-me-up",
   "version": "1.0.0",
-  "dependencies": {}
+  "scripts": {
+    "dev": "nodemon --exec ts-node src/main.ts",
```

Comment:
taki hint, choć może już wiesz:
Zapis `--exec ts-node` nie jest potrzebny bo jak jest `ts-node` i wykonujesz plik `*.ts` to on sobie to "z automatu" ogarnia.